gradle/jdks/11/linux-glibc/aarch64/download-url

```diff
@@ -1 +0,0 @@
-https://corretto.aws/downloads/resources/11.0.26.4.1/amazon-corretto-11.0.26.4.1-linux-aarch64.tar.gz
```

Comment:
deleted by `./gradlew setupJdks` because we are not using java 11 now anymore: https://github.com/palantir/gradle-conjure/pull/1598/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R85